docs/docsource/operation/account_operations/create_access_keys.rst

```diff
@@ -0,0 +1,59 @@
+.. _create_access_keys:
+
+Create Access Keys
+==================
+
+Create Access Keys Using the UI
+-------------------------------
+
+#. Log into the |product| UI using previously provisioned credentials (refer to :ref:`Create an Account`).
+
+#. Click **Accounts** on the top menu.
+
+   .. image:: ../../graphics/account_banner.PNG
+
+#. Click **+ Create Access key**
+
+   .. image:: ../../graphics/accounts-page.PNG
+
+#. Click **-> Continue** on the **Create Root user Access keys** screen.
+
+   .. image:: ../../graphics/root_user_access_keys_dialog.PNG
+
+#. Save the Secret Access key ID by clicking the copy button, and save it in a secure location 
+   
+   .. image:: ../../graphics/close_button.PNG
+
+#. Close **Close** after you saved the Secret Access key ID.
+
+Create Access Keys Using the Command Line
+-----------------------------------------
+
+#. Set the admin access key ID and secret access key ID as environment variables.
+
+   .. code::
```

Comment:
I'd put these commands in [code-blocks](https://docs.google.com/document/d/1_N_-yrRGiygMxdJVhKdI8tEgCck50aNyH6UJsuEAnL4/edit#heading=h.kads838eo5z) for clarity. 
Caption 1 would be sth like `Vault Container`. Caption 2= `Admin Access key ID`. Caption 3= `Secret Access key ID`.